sdks/python/apache_beam/yaml/standard_io.yaml

```diff
@@ -112,3 +112,38 @@
     'WriteToJson': 'beam:schematransform:org.apache.beam:json_write:v1'
   config:
     gradle_target: 'sdks:java:extensions:schemaio-expansion-service:shadowJar'
+
+- type: renaming
+  transforms:
+    'ReadFromJdbc': 'ReadFromJdbc'
+    'WriteToJdbc': 'WriteToJdbc'
+  config:
+    mappings:
+      'ReadFromJdbc':
+        driver_class_name: 'driverClassName'
+        jdbc_url: 'jdbcUrl'
+        username: 'username'
+        password: 'password'
+        table_name: 'location'
```

Comment:
Are table_name and read_query mutually exclusive? (Should we just call them table and query for symmetry with BQ?)

Reply:
I agree. 

WDYT about `url` instead of `jdbc_url`?

Reply:
+1